experiments/openebs/openebs-pool-container-failure/cstor_pool_kill.yml

```diff
@@ -1,2 +1,4 @@
 - include: cstor_kill_and_verify_pool.yml
-  loop: "{{ range(0, chaos_iterations|int, 1)|list }}"
\ No newline at end of file
+  loop: "{{ range(0, chaos_iterations|int, 1)|list }}"
+  loop_control:
+    loop_var: outer_item
```

Comment:
Are there any similar usecases where we have to use loop_var? Or where use of latter is preferred over current model.

Reply:
No, I think this is the only place which is conflicting with our new changes ( job to pod conversion)